TODO.md

```diff
@@ -0,0 +1,32 @@
+# CNCF Hugo Starter Checklist
+
+## CNCF Website Guidelines
+
+As per the [CNCF Website Guidelines](https://github.com/cncf/foundation/blob/master/website-guidelines.md):
+
+- [ ] 1. Website should be hosted in an open source repo
+  - [ ] Hosted in the same organization as the main project
+- [ ] 2. Reference the origin company correctly
+  *Note*: It is OK to say that, e.g., “Prometheus was originally created by Soundcloud” or “Kubernetes builds upon 15 years of experience of running production workloads at Google,” but the origin company should not otherwise be referred to on the project homepage.
+- [ ] 3. No links or forms for capturing enterprise support leads present.
+  It is fine to have an enterprise support, commercial partners or similar page.
+  - [ ] If page is present, the companies list is alphabetized or randomized on load.
+  - [ ] If page is present, the vetting of companies listed is complete
+    *Note*: Projects are welcome to outsource this vetting to CNCF staff if it becomes a burden.
+- [ ] 4. Links to companies offering support go to a page that at least mentions support of the project
+- [ ] 5. Copyright notice present at bottom of page.
+  Copyright to the project authors or to CNCF, not the origin company.
```

Comment:
If we're going to add this as a todo, we owe it to projects to give them the specific licensing terminology we use:

```
© startYear-currentYear projectName Authors | Documentation Distributed under CC-BY-4.0

© currentYear The Linux Foundation. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our [Trademark Usage](https://www.linuxfoundation.org/trademark-usage/) page.
```

Also note: this starter _should_ include a shortcode which auto-populates said trademark footer correctly. But if not, it would be helpful if you opened an issue 🙏

Reply:
Opened an issue here: https://github.com/cncf/hugo-netlify-starter/issues/34